### PR TITLE
[ADD]: PostgreSQL Backend for Workloads Backend

### PR DESCRIPTION
**Pull request by bhaskarhc**
# :: PostgreSQl Statefulset Workload Backend  :hammer_and_pick:
### Install postgreSQL Package to make DB connection  

```sh
$ npm install pg --save
```

### what's this code does :grey_question:  

- >Connecting Postgresql DB using service name **"pgset"** Running in GKE .
- >Saving multiple user data , Reading Data in DB .

**Deployment :https://github.com/openebs/openebs/tree/master/k8s/demo/crunchy-postgres**  .

### [ Fix ]:Cockroachdb

- Creating Database maya .
- Fixed order of creating Connection :arrow_forward: Db  :arrow_forward:  Table :checkered_flag:

### CODE REVIEW

File: database/postgresql-jiva/dbconnection.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var config = {
   user: "testuser",
   database: "userdb",
   password: "password",
-  host: "pgset.postgresql-jiva",
+  host: "pgset-primary.postgresql-jiva",
```
Comment (inyee786) [inline database/postgresql-jiva/dbconnection.js]: it should be pgset.postgresql-jiva

Comment (bhaskarhc) [reply database/postgresql-jiva/dbconnection.js]: Pgset-primary is also service for postgresql

Comment (inyee786) [reply database/postgresql-jiva/dbconnection.js]: it will not work, did you tested it?

Comment (bhaskarhc) [reply database/postgresql-jiva/dbconnection.js]: No , i didnt test ! In local I can’t test by using service ! 

In deployment repo using this host name.!

I will change it to ‘pgset’ 👍👍

**Comment by inyee786** [comment]
@bhaskarhc  can you deploy the cockroach on minikube in one namespaces and the ui and backend component in another namespace and check whether the data are get written on the cockroachdb, and share the screenshot


checkout the below link for more information

https://www.cockroachlabs.com/docs/stable/orchestrate-cockroachdb-with-kubernetes.html
https://github.com/openebs/openebs/blob/master/k8s/demo/cockroachDB/cockroachdb-lg.yaml

**Comment by chandankumar4** [comment]
@bhaskarhc Please specify proper commit message and merge all commit into one!

**Comment by inyee786** [comment]
@bhaskarhc can you follow the below code example for creating the connection then database and the table

 
```
var async = require('async');
var fs = require('fs');
var pg = require('pg');

// Connect to the "bank" database.
var config = {
    user: 'maxroach',
    host: 'localhost',
    database: 'bank',
    port: 26257
};

// Create a pool.
var pool = new pg.Pool(config);

pool.connect(function (err, client, done) {

    // Close communication with the database and exit.
    var finish = function () {
        done();
        process.exit();
    };

    if (err) {
        console.error('could not connect to cockroachdb', err);
        finish();
    }
    async.waterfall([
            function (next) {
                // Create the 'accounts' table.
                client.query('CREATE TABLE IF NOT EXISTS accounts (id INT PRIMARY KEY, balance INT);', next);
            },
            function (results, next) {
                // Insert two rows into the 'accounts' table.
                client.query('INSERT INTO accounts (id, balance) VALUES (1, 1000), (2, 250);', next);
            },
            function (results, next) {
                // Print out account balances.
                client.query('SELECT id, balance FROM accounts;', next);
            },
        ],
        function (err, results) {
            if (err) {
                console.error('Error inserting into and selecting from accounts: ', err);
                finish();
            }

            console.log('Initial balances:');
            results.rows.forEach(function (row) {
                console.log(row);
            });

            finish();
        });
});
```

**Review by inyee786 — APPROVED** [review]
LGTM

**Pull request merged**